warehouse/ingest-core/src/test/resources/MarkingFunctionsContext.xml

```diff
@@ -18,7 +18,7 @@
 
 	<!-- only properties that you want re-mapped or excluded need to be included in the map -->
 	<!-- mapping to empty string will exclude that property from the markingFunctions results -->
-    <bean id="markingFunctions" class="datawave.marking.MarkingFunctions$NoOp" />
+    <bean id="markingFunctions" class="datawave.marking.MarkingFunctions.Default" />
```

Comment:
should this be $Default ?

Reply:
A quick test loading the file in a spring context shows that it works the same either way.
I'll add a unit test to prove it tomorrow.